images/win/scripts/Installers/Windows2019/Install-VS2019.ps1

```diff
@@ -151,21 +97,19 @@ $WorkLoads = '--allWorkloads --includeRecommended ' + `
               '--add Microsoft.VisualStudio.Workload.VisualStudioExtension'
 
 
-$ReleaseInPath = 'Enterprise'
-$Sku = 'Enterprise'
-$VSBootstrapperURL = 'https://aka.ms/vs/16/release/vs_Enterprise.exe'
-
-$ErrorActionPreference = 'Stop'
+$ReleaseInPath = "Enterprise"
+$BootstrapperName = "vs_${ReleaseInPath}.exe"
+$BootstrapperUrl = "https://aka.ms/vs/16/release/${BootstrapperName}"
 
 # Install VS
-$exitCode = InstallVS -WorkLoads $WorkLoads -Sku $Sku -VSBootstrapperURL $VSBootstrapperURL
+Install-VS -WorkLoads $WorkLoads -BootstrapperName $BootstrapperName -BootstrapperUrl $BootstrapperUrl
```

Comment:
Arguments order is different in VS17 and VS19

Reply:
Done. Thanks!